File: back-end/src/main/java/com/csulb/cookie/domain/Merchant.java
package com.csulb.cookie.domain;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import lombok.Data;

@TableName(value ="merchant")
@Data
public class Merchant implements Serializable {

    @TableId(type = IdType.AUTO)
    private Integer id;
    private String username;
    private String password;
    private String email;
    private String phone;
    private String shopName;
    private String description;
    private String businessHour;

    @TableField(exist = false)
    private String street;

    @TableField(exist = false)
    private String city;

    @TableField(exist = false)
    private String state;

    @TableField(exist = false)
    private Integer zip;


}

File: front-end/lib/domain/merchant.dart
class Merchant {
  String username;
  String password;
  String email;
  String phone;
  String shopName;
  String description;
  String businessHour;
  String street;
  String city;
  String state;
  int zip;

  Merchant(
      this.username,
      this.password,
      this.email,
      this.phone,
      this.shopName,
      this.description,
      this.businessHour,
      this.street,
      this.city,
      this.state,
      this.zip);

  @override
  String toString() {
    return 'Merchant{username: $username, password: $password, email: $email, phone: $phone, shopName: $shopName, description: $description, '
        'businessHour: $businessHour, street: $street, city: $city, state: $state, zip: ${zip.toString()}}';
  }
}